tutorials/nservicebus-101/lesson-2/Snippets/Core_6/ComplexCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using NServiceBus;
+
+namespace Core_6
+{
+    #region ComplexCommand
+
+    public class DoSomethingComplex :
+        ICommand
+    {
+        public int SomeId { get; set; }
+        public ChildClass ChildStuff { get; set; }
```

Comment:
@weralabaj After reading this several times and being confused I finally realized the disconnect. I meant that if you implement the ICommand or IEvent interfaces, not implementing messages AS interfaces.

For the purposes of the tutorial I'm sticking with classes for sure.